Aurora12/ALARM_AND_ABORT.agc

```diff
@@ -25,3 +25,98 @@
 #       https://www.ibiblio.org/apollo.  
 # The original high-quality digital images are available at archive.org:
 #       https://archive.org/details/aurora00dapg
+
+## PAGE 336
```

Comment:
For future submissions, use "Page" instead of "PAGE". We reserve the all-caps for stuff originally in the listings, and add lowercase for things we add ourselves.